hypha/apply/funds/tables.py

```diff
@@ -262,8 +266,9 @@ class SubmissionFilter(filters.FilterSet):
     lead = Select2ModelMultipleChoiceFilter(queryset=get_round_leads, label='Leads')
     reviewers = Select2ModelMultipleChoiceFilter(queryset=get_reviewers, label='Reviewers')
     screening_statuses = Select2ModelMultipleChoiceFilter(queryset=get_screening_statuses, label='Screening', null_label='No Status')
+    form_data = Select2ModelMultipleChoiceFilter(queryset=get_categories, label='Categories')
```

Comment:
This obviously do not work at all but I'm hopeful you can figure out how to do it.